cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,118 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let worker;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('addWorker').then((addWorker) => {
```

Comment:
The worker object generated by the addWorker task should be unique for each test. Currently, you generate a worker only once before the beforeEach hook, and then reuse the same worker for all test cases. This may cause issues, especially in the case of the "Delete all workers" test, where multiple workers are deleted. It would be better to generate a new worker for each test.